cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/strategy/PostponeUrpExecutionTaskStrategy.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2018 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+
+package com.linkedin.kafka.cruisecontrol.executor.strategy;
+
+import com.linkedin.kafka.cruisecontrol.executor.ExecutionProposal;
+import com.linkedin.kafka.cruisecontrol.executor.ExecutionTask;
+import java.util.Collection;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.SortedSet;
+import java.util.TreeSet;
+import org.apache.kafka.common.Cluster;
+import org.apache.kafka.common.PartitionInfo;
+
+
+public class PostponeUrpExecutionTaskStrategy implements ExecutionTaskStrategy {
+
+  private boolean isDataMoveOnUrp(Cluster cluster, ExecutionTask task) {
```

Comment:
We have the same logic at multiple places (e.g. `PreferredLeaderElectionGoal`, `KafkaClusterState`). Can we move it to some common util function and use it for such checks?